packages/web-pkg/tests/unit/components/CreateLinkModal.spec.ts

```diff
@@ -30,64 +30,26 @@ const selectors = {
   contextMenuToggle: '#link-modal-context-menu-toggle',
   confirmBtn: '.link-modal-confirm',
   cancelBtn: '.link-modal-cancel',
-  linkRoleDropDownToggle: '.link-role-dropdown-toggle',
-  modalAdvancedModeButton: '[data-testid="modal-advanced-mode-button"]'
+  linkRoleDropDownToggle: '.link-role-dropdown-toggle'
 }
 
 describe('CreateLinkModal', () => {
   describe('password input', () => {
-    it('should not rendered when "isAdvancedMode" is not set', async () => {
+    it('should be rendered', () => {
       const { wrapper } = getWrapper()
-      await nextTick()
-      expect(wrapper.find(selectors.passwordInput).exists()).toBeFalsy()
-    })
-    it('should be rendered', async () => {
-      const { wrapper } = getWrapper()
-      wrapper.find(selectors.modalAdvancedModeButton).trigger('click')
-      await nextTick()
       expect(wrapper.find(selectors.passwordInput).exists()).toBeTruthy()
     })
-    it('should not be rendered if user cannot create public links', () => {
```

Comment:
Dropping these tests as they seem to have been false positives. Permissions do not have any effect on the visibility of those fields.